tests/deliver/test_deliver.py

```diff
@@ -131,26 +131,24 @@ def test_update_sample_delivery(self):
         with self.assertRaises(NotImplementedError):
             self.deliverer.update_delivery_status()
 
-    @mock.patch.object(
-        deliver.db.db.CharonSession,
-        'project_create',
-        return_value="mocked return value")
+    @mock.patch('taca_ngi_pipeline.deliver.deliver.db.db.CharonSession', autospec=taca_ngi_pipeline.deliver.deliver.db.db.CharonSession)
```

Comment:
Double db as well?

Reply:
The first db is `taca_ngi_pipeline.utils.database` which is imported as `db` in `taca_ngi_pipeline.deliver.deliver` and the second is `ngi_pipeline.databases.classes` which is imported as `db` in `taca_ngi_pipeline.utils.database`. Its quite the journey.